module/notifications/push/helper.js

```diff
@@ -623,8 +623,11 @@ async function _sendMessage(notificationInformation) {
         try {
 
             let deviceId = notificationInformation.token;
+            let appType = false;
+            if(notificationInformation.data.appType &&  notificationInformation.data.appType != undefined){
```

Comment:
@priyanka-TL , @aman-tunerlabs - When will appType not be passed?

Reply:
@aks30  it will happen sometimes because i got an issue in importFromProjectTemplate there not aware of appType so they are not passing appType its triggered an error. dont know where ever we are using push notification if they dont send appType we need to handle

Reply:
@aman-tunerlabs - Any idea if that is happening, ideally it should not happen, right?

Cc - @priyanka-TL

Reply:
@priyanka-TL It should not happen . Please make a change as mentioned on comment.